galette/lib/Galette/Core/Galette.php

```diff
@@ -207,13 +207,22 @@ public static function getMenus(bool $public = false): array
                 || $login->isStaff()
                 || ($login->isGroupManager() && $preferences->pref_bool_groupsmanagers_create_member)
             ) {
-                $menus['members']['items'][] = [
-                    'label' => _T("Add a member"),
-                    'title' => _T("Add new member in database"),
-                    'route' => [
-                        'name' => 'addMember'
+                $menus['members']['items'] = array_merge($menus['members']['items'], [
+                    [
+                        'label' => _T("Add a member"),
+                        'title' => _T("Add new member in database"),
+                        'route' => [
+                            'name' => 'addMember'
+                        ]
+                    ],
+                    [
+                        'label' => _T("Empty adhesion form"),
+                        'title' => _T("Download empty adhesion form"),
+                        'route' => [
+                            'name' => 'emptyAdhesionForm'
+                        ]
```

Comment:
As this is not an authenticated route, what do you think about adding a link on unauthenticated pages ? (with a setting to let admin choose to display this link or not :sunglasses:)

Reply:
> As this is not an authenticated route, what do you think about adding a link on unauthenticated pages ? (with a setting to let admin choose to display this link or not 😎)

See #606

Reply:
Yes, that can make sense, thank you :)